client/DeckStatus.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DeckStatusSummary from './DeckStatusSummary';
+import StatusPopOver from './StatusPopOver.jsx';
+
+class DeckStatus extends React.Component {
+    render() {
+        let { status } = this.props;
+        let statusName;
+        let className = 'deck-status';
+
```

Comment:
Surprised that eslint doesn't pick up this double space, will have to see if I can find a rule for that